src/Form/AdminSettingsForm.php

```diff
@@ -106,6 +107,19 @@ private function buildCredentialForm() {
       '#default_value' => $credential_settings['site_id'],
       '#required' => TRUE,
     ];
+    $form['public_key'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Public Key'),
+      '#description' => 'Your Acquia Lift Public Key.',
+      '#default_value' => $credential_settings['public_key'],
+      '#required' => TRUE,
+    );
+    $form['secret_key'] = array(
+      '#type' => 'password',
+      '#description' => 'Your Acquia Lift Secret Key. Only required when setting it up for the first time or when you are changing the secret key.',
+      '#title' => t('Secret Key'),
+      '#default_value' => $credential_settings['secret_key'],
+    );
```

Comment:
Looked at 8.x-1.x and copied the few lines that were added to the secret key and public key. There was a conditional description but I kept it static. I could not see the value add of making that conditional